### ISSUE TITLE
`python.django.performance.access-foreign-keys.access-foreign-keys` doesn't exclude `request.user.id`

### ISSUE THREAD

**Issue by fgarci03** [returntocorp/sgrep-rules#1644]
**Describe the bug**
In Django, the views get a request instance that contains a User object. Since it's common to access `request.user.id`, and the request object is NOT a model instance, this rule triggers a false positive that could perhaps be ignored with `pattern-not: request.user.id`

**To Reproduce**
```python
from django.http import HttpResponse

def cool_view(request):
    return HttpResponse({'user_id': request.user.id})
```

**Expected behavior**
This example should not trigger this rule, since "request" is not a model instance

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [X] P2: annoying but not blocking me

**Additional Context**
N/A

**Comment by fgarci03** [comment]
I just noticed this also happens when doing `self.user.id` :scream:
